tools/c7n_azure/c7n_azure/utils.py

```diff
@@ -37,3 +37,16 @@ def utcnow():
     """The datetime object for the current time in UTC
     """
     return datetime.datetime.utcnow()
+
+
+class Math(object):
```

Comment:
Especially for utility stuff that will get used a lot lets make sure we have unit tests.  There is already a test_azure_utils.py so should be trivial to add a few more tests in for these methods.

Reply:
tx - resolved